src/components/shared/AlertBox.tsx

```diff
@@ -92,6 +94,17 @@ const AlertBox = forwardRef<any, Props>(function NavLinkRef(
                 '& > .MuiAlert-message': {
                     p: 1,
                 },
+                '& > .MuiAlert-action': short
+                    ? {
+                          margin: 0,
+                          padding: 1,
+                          paddingTop: 0.5,
+                          alignItems: 'center',
+                      }
+                    : {
+                          margin: 0,
+                          padding: 1,
+                      },
```

Comment:
Styling to make the close button be centered for the short and at the top for the large version. Still not perfect but okay.